server/db.js

```diff
@@ -1,112 +1,184 @@
-const Sequelize = require('sequelize');
-// const Seed = require('../seed.js');
-const { username, password } = require('../config.js');
+const { Pool, Client } = require('pg');
```

Comment:
Can remove Client because you aren't using it anywhere. Pool should be fine alone anyway.

Reply:
Done!